headers/types/common/kaomado.h

```diff
@@ -0,0 +1,54 @@
+// Represents the state of a portrait that wants to be displayed

```

Comment:
I guess I'm okay with adding a separate `kaomado.h` for structs, but there needs to be a `#include "kaomado.h"` at the bottom of `common.h` so that it's reachable from `pmdsky.h`.

This file also needs an include guard:

```c
#ifndef HEADERS_TYPES_COMMON_KAOMADO_H_
#define HEADERS_TYPES_COMMON_KAOMADO_H_

...

#endif
```